app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(commands: str) -> None:
+    command, original_file, copy_file = commands.split()
+    if command == "cp" and original_file != copy_file:
+        content = open(original_file, "r")
```

Comment:
ur file won't be closed
![image](https://user-images.githubusercontent.com/103201010/220115042-3cc1738c-ddb4-4d0f-8f36-6140ddd6c0f4.png)